modules/examples/lnl.nix

```diff
@@ -125,7 +125,7 @@
     set -g pane-border-style fg=black
     set -g status-bg black
     set -g status-fg white
-    set -g status-right '#[fg=white]#(id -un)@#(hostname)   #(cat /run/current-system/darwin-version)'
+    set -g status-right '#[fg=white]#(id -un)@#(hostname)   #(jq --raw-output '.darwinLabel' /run/current-system/darwin-version.json)'
```

Comment:
This `jq` invocation seems to be ~15 times more expensive than a `cat`, which seems unfortunate for a shell prompt. I guess it's probably still cheaper than something like `git`, but maybe we could continue exposing `/run/current-system/darwin-version` (or rename it to `darwin-label`)? It's not my prompt though so I don't care that much ultimately I guess :)

Reply:
From my limited searching on GitHub, it doesn't seem like many people depend on the existing `/run/current-system/darwin-version` file, so I've gone ahead and merged it as is, however if more people report wanting `darwin-label` as a separate file, I'm happy to add it back